src/item_factory.cpp

```diff
@@ -1491,6 +1491,13 @@ void Item_factory::load( islot_comestible &slot, JsonObject &jo, const std::stri
             }
         }
     }
+
+    if( jo.has_string( "rot_spawn" ) ) {
+        slot.rot_spawn = jo.get_string( "rot_spawn" );
+    }
+    if( jo.has_string( "rot_spawn_chance" ) ) {
```

Comment:
If you use `assign` to load the value, you don't need to check for existance of that member. `assign` will handle that case on its own (and do nothing).